src/res_printer.ml

```diff
@@ -99,6 +99,15 @@ let hasCommentBelow tbl loc =
   | [] -> false
   | exception Not_found -> false
 
+let rec hasNestedJsx expr =
+  match expr.Parsetree.pexp_desc with
+  | Pexp_construct ({txt = Longident.Lident "[]"}, _) -> false
+  | Pexp_construct
+      ( {txt = Longident.Lident "::"},
+        Some {pexp_desc = Pexp_tuple (hd :: [tail])} ) ->
+    hasNestedJsx hd || hasNestedJsx tail
```

Comment:
Should this be `ParsetreeViewer.isJsxExpression hd` ? and the next line do nothing? (and just return false)

Reply:
Yes. Otherwise it behaves differently than before this commit (see the spread change in this commit which I think we do not want).